src/MongoDB.Driver.Core/Core/Authentication/DefaultAuthenticator.cs

```diff
@@ -36,6 +36,7 @@ public class DefaultAuthenticator : IAuthenticator
         private readonly UsernamePasswordCredential _credential;
         private readonly IRandomStringGenerator _randomStringGenerator;
 
+        private ScramSha256Authenticator _speculativeScramSha256Authenticator;
```

Comment:
Fix whitespace and refactor to:

    private IAuthenticator _speculativeAuthenticator;

Reply:
Oops, I was separating the fields by readonly and non-readonly: fixed the whitespace and took your refactor.